doc/content/getting_started/installation.md

```diff
@@ -112,15 +113,25 @@ mamba update --all
 ```
 
 This assumes that your MOOSE-based development environment is named `moose`. To update your TMAP8
-repository, use the following commands.
+repository as a TMAP8 user, use the following commands, which provide general users the most stable
+branch (`upstream/main`):
 
 ```bash
 cd ~/projects/TMAP8
 git fetch upstream
```

Comment:
```suggestion
cd ~/projects/TMAP8
git checkout main
git fetch upstream
```

Let's also start requesting a checkout here to make sure they are updating against the right local branch.